content/en/docs/advanced-tasks/configuring-olm.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Configuring OLM"
+linkTitle: "Configuring OLM"
+weight: 3
+---
+
+# Configuring OLM
+
+The [Operator-lifecycle-Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager) controller can be configured through an  [OLMConfig CustomResourceDefinition (CRD)](https://github.com/operator-framework/operator-lifecycle-manager/blob/3f3f00644ca4a50d845db2b97c12afcea56fd5d8/deploy/chart/crds/0000_50_olm_00-olmconfigs.crd.yaml) named `cluster`. This document will outline what configurations OLM currently supports.
+## Disabling Copied CSVs
+
+### Background
+When an Operator is installed by OLM, a stripped down copy of its CSV is created in every namespace the Operator is configured to watch. These stripped down CSVs are known as "Copied CSVs" and communicate to users which controllers are actively reconciling resource events in a given namespace. When operators are installed in the AllNamespace mode, a Copied CSV is created in every namespace on the cluster. On especially large clusters, with namespaces and installed operators tending in the hundreds or thousands, Copied CSVs consume an untenable amount of resources; e.g. OLM's memory usage, cluster Etcd limits, networking, etc.
```

Comment:
nit/feel free to ignore: newline between the Background header and the content to suppress any markdown linting violations locally.

Reply:
Done